index.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable max-classes-per-file */
+import { Book, Store, showBooks } from './modules/class.js';
+import { DateTime } from './node_modules/luxon/build/es6/luxon.js';
+
+document.addEventListener('DOMContentLoaded', showBooks.displayBooks);
+
+const showBookList = document.getElementById('nav-list');
+const showAddBook = document.getElementById('nav-add');
+const showContact = document.getElementById('nav-contact');
+
+function showList() {
+  document.getElementById('book-list').style.display = 'flex';
+  document.getElementById('add-new').style.display = 'none';
+  document.getElementById('contact').style.display = 'none';
+}
+function showAdd() {
+  document.getElementById('book-list').style.display = 'none';
+  document.getElementById('add-new').style.display = 'flex';
+  document.getElementById('contact').style.display = 'none';
+}
+function showContactPage() {
+  document.getElementById('book-list').style.display = 'none';
+  document.getElementById('add-new').style.display = 'none';
+  document.getElementById('contact').style.display = 'flex';
+}
```

Comment:
- [ ] Kindly use `ES6` syntax for creating a function. It looks like this `const showContactPage = () => {some code}`